### PR TITLE
Builddir split

### PR DESCRIPTION
**Pull request by dpsmith**
This PR enables splitting the build base from the build directory.

### CODE REVIEW

File: bordel [hunk 1]
```diff
@@ -26,9 +24,15 @@ cd ${TOP}
 register_cmds ${BORDEL_DIR}/cmds
 
 # Parse options.
-while getopts ":i:h" opt; do
+while getopts ":b:i:h" opt; do
     case $opt in
         h)  usage 0 ;;
+        b)  BASE_DIR="$( cd "$( realpath ${OPTARG} )" >/dev/null 2>&1 && pwd )"
+            if [ ! -d ${BASE_DIR} ]; then
```
Comment (eric-ch) [inline bordel:31]: `$()` does not require quotes, but variables do (space in path, etc). There are a few others and some are out of this changeset.

Comment (dpsmith) [reply bordel:31]: Reused from BORDEL_DIR which was very carefully crafted but I can drop here.

**Review by eric-ch — APPROVED** [review]
The only change I saw trying this is that some symlink to relative path became absolute (`BASE_DIR`).
It shouldn't be a problem and it is documented.

**Comment by eric-ch** [comment]
That PR got forgotten for a while.
Merging soon.

**Pull request merged**